corehq/apps/locations/permissions.py

```diff
@@ -319,6 +319,19 @@ def user_can_access_any_location_id(domain, user, location_ids):
             .exists())
 
 
+def user_can_access_other_user(domain, user, other_user):
+    if user.has_permission(domain, 'access_all_locations'):
+        return True
+
+    accessible_location_ids = (SQLLocation.objects .accessible_to_user(domain, user) .values_list('id', flat=True))
```

Comment:
ah oops i didn't mean to commit that. it was originally on multiples lines. i just hit `J` in vim 3 times and then copy and paste to ipdb to debug. then i'm supposed to format it back...